src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/discount_list.html.twig

```diff
@@ -22,28 +22,30 @@
  * @copyright Since 2007 PrestaShop SA and Contributors
  * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
  *#}
-<table class="table discountList{% if discounts is empty %} d-none{% endif %}">
-  <thead>
-  <tr>
-    <th>{{ 'Name'|trans({}, 'Admin.Global') }}</th>
-    <th>{{ 'Value'|trans({}, 'Admin.Global') }}</th>
-    <th class="text-right d-print-none">{{ 'Actions'|trans({}, 'Admin.Global') }}</th>
-  </tr>
-  </thead>
-  <tbody>
-  {% for discount in discounts %}
+<div class="col-md-12">
+  <table class="table discountList{% if discounts is empty %} d-none{% endif %}">
+    <thead>
     <tr>
-      <td class="discountList-name">{{ discount.name }}</td>
-      <td>
-        {% if discount.amountRaw.greaterThan(number(0)) %}-{% endif %}
-        {{ discount.amountFormatted }}
-      </td>
-      <td class="text-right d-print-none">
-        <a class="delete-cart-rule btn btn-text" href="{{ path('admin_orders_remove_cart_rule', {'orderId': orderId, 'orderCartRuleId': discount.orderCartRuleId}) }}">
-          <i class="material-icons">delete</i>
-        </a>
-      </td>
+      <th>{{ 'Name'|trans({}, 'Admin.Global') }}</th>
+      <th>{{ 'Value'|trans({}, 'Admin.Global') }}</th>
+      <th class="text-right d-print-none">{{ 'Actions'|trans({}, 'Admin.Global') }}</th>
     </tr>
-  {% endfor %}
-  </tbody>
-</table>
+    </thead>
+    <tbody>
+    {% for discount in discounts %}
+      <tr>
+        <td class="discountList-name">{{ discount.name }}</td>
+        <td>
+          {% if discount.amountRaw.greaterThan(number(0)) %}-{% endif %}
+          {{ discount.amountFormatted }}
+        </td>
+        <td class="text-right d-print-none">
+          <a class="delete-cart-rule btn btn-text" href="{{ path('admin_orders_remove_cart_rule', {'orderId': orderId, 'orderCartRuleId': discount.orderCartRuleId}) }}">
+            <i class="material-icons">delete</i>
+          </a>
+        </td>
+      </tr>
+    {% endfor %}
```

Comment:
Indendation is wrong, you should add indents to the for

Reply:
Mmm, I didn't change anything from the previous indent... just added one level

But what do mean? putting the `for` on the same level as the `tr`? I'm not so sure, keeping a different level from the loop and its content helps to identify it IMHO Besides this way it also respects the HTML indentation compared with the parent `tbody`

Reply:
I mean, add indents in order to avoir having the for at the same level then then tbody, like you said it improve readability, don't you think?

I see the `for` as a child of the tbody myself, but maybe we should keep it..

Reply:
Indeed we don't have clear "code style" for this I think it's better to keep a difference between the loop statements and its content to see clearly where the loop start and ends But that's debatable 🤷‍♂️

I'm not sure it's worth blocking the PR though especially since it's a critical issue, but it's worth discussing it to avoid further questions in the future